highfidelity/hifi
# makeUserConnection.js Fix for alternative skeletons

Allows avatars which have an alternative bone structure to make connections. (Hands are still required)

Fixes https://highfidelity.fogbugz.com/f/cases/4360/handshake-fails-for-non-standard-avatars-that-don-t-have-Left-RightHandMiddle1-joint

## QA Test
- Test with two avatars.
- Test with and without HMD.
- One avatar should use this avatar http://mpassets.highfidelity.com/1b7e1e7c-6c0b-4f20-9cd0-1d5ccedae620-v1/bb64e937acf86447f6829767e958073c.fst
- Try to make a connection using the X key.
- Try to make a connection using the grip button on your handcontroller.
